### ISSUE TITLE
Fans On Auto with User-Defined Min Speed

### ISSUE THREAD

**Issue by jasonmvp** [crystalidea/macs-fan-control#312]
I have a new Mac Pro 7,1 with the Pro version of MFC.  I'd like to set Fan #2 to a slightly higher baseline/min speed, but I still want MacOS to spin it up automatically if it needs to.  Is that how the setting works?  Or is it something you could add to the app?

Thanks!

**Comment by zecanard** [comment]
Similar #1 and to #256.

**Comment by jasonmvp** [comment]
I guess setting it "Sensor based" is a good alternative.  I'm just not certain which one to set to.  The key thing I'm trying to cool are the NVMe drives in my 2 HighPoint PCI-E cards  They're sitting right smack behind Fan #2, so I just wanted to boost that one up a bit under normal circumstances.

So, for instance: set the fan as a min of 600RPMs and then let SMC handle the control from there.  Is that doable?  Unfortunately the HighPoint cards don't allow the drives' SMART info to get to the OS.  The only way to see their individual temperatures is via the HighPoint utility.  :-(

**Comment by zecanard** [comment]
No, I don’t think that’s doable right now, and I agree that it would be nice to have more granular/complex fan controls.
